.github/ISSUE_TEMPLATE/bug-report.yaml

```diff
@@ -0,0 +1,65 @@
+name: Bug Report
+description: Report a bug encountered with Scylla Operator
+labels: kind/bug
+body:
+- type: textarea
+  id: problem
+  attributes:
+    label: What happened?
+    description: |
+      Please provide as much info as possible. Not doing so may result in your bug not being addressed in a timely manner.
```

Comment:
I'd reword it to sound more like an encouragement than a threat. ;) How about "Please provide as much information as possible. Ensuring all details are shared will help us address your bug in a timely manner."

Reply:
also identical to Kube but rephrased